docs/reference/decision_points/utility.md

```diff
@@ -7,6 +7,42 @@
     Utility is a combination of [Automatable](./automatable.md) and
     [Value Density](./value_density.md)
 
+
+This is a compound decision point, therefore it is a notational convenience.
+
+*Utility* estimates an adversary's benefit compared to their effort based on the assumption that they can exploit the vulnerability.
+*Utility* is independent from the state of [*Exploitation*](exploitation.md), which measures whether a set of adversaries have ready access to exploit code or are in fact exploiting the vulnerability.
+
+*Utility* estimates an adversary's benefit compared to their effort based on the assumption that they can exploit the vulnerability.
```

Comment:
Is this intentionally a repetition of line 13?

Reply:
Oof. Bad file merge on my side. I'll fix this and the next one in an update to the branch next week.

Reply:
Fixed in 785f928